File: app/src/main/java/caribouapp/caribou/com/cariboucoffee/cybersource/TokenResponse.java
package caribouapp.caribou.com.cariboucoffee.cybersource;

import java.io.Serializable;

import caribouapp.caribou.com.cariboucoffee.mvp.account.model.CardEnum;

/**
 * Created by andressegurola on 12/4/17.
 */

public class TokenResponse implements Serializable {
    private String mPaymentToken;

    private CardEnum mCardType;

    private String mMaskedCardNumber;

    public String getPaymentToken() {
        return mPaymentToken;
    }

    public void setPaymentToken(String paymentToken) {
        this.mPaymentToken = paymentToken;
    }

    public CardEnum getCardType() {
        return mCardType;
    }

    public void setCardType(CardEnum cardEnum) {
        mCardType = cardEnum;
    }

    public String getMaskedCardNumber() {
        return mMaskedCardNumber;
    }

    public void setMaskedCardNumber(String maskedCardNumber) {
        mMaskedCardNumber = maskedCardNumber;
    }
}

File: app/src/main/java/caribouapp/caribou/com/cariboucoffee/fiserv/model/TokenResponse.kt
package caribouapp.caribou.com.cariboucoffee.fiserv.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Swapnil on 03/10/22.
 */
class TokenResponse {
    @SerializedName("tokenType")
    var tokenType: String? = null
        private set

    @SerializedName("tokenId")
    var tokenId: String? = null
        private set

    @SerializedName("tokenProvider")
    var tokenProvider: String? = null
        private set
}
